Command/LoadYamlFixturesCommand.php

```diff
@@ -66,8 +65,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if ($input->isInteractive() && !$input->getOption('append')) {
-            /* @var DialogHelper $dialog */
-            $dialog = $this->getHelperSet()->get('dialog');
+            $helperName = class_exists('Symfony\Component\Console\Helper\QuestionHelper') ? 'question' : 'dialog';
```

Comment:
@tomas-pecserke another possible solution would be to use `SymfonyStyle::confirm`

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Style/SymfonyStyle.php#L230

It can be used via

``` php
$symfonyStyle = new SymfonyStyle($input, $output);
$symfonyStyle->confirm('Careful, database will be purged. Do you want to continue Y/N ?', false);
```